src/Core/Customer/Application/Resolver/CreateCustomerMutationResolver.php

```
@@ -0,0 +1,79 @@
+     *             initials: string,
+     *             email: string,
+     *             phone: string,
+     *             leadSource: string,
+     *             type: string,
+     *             status: string,
+     *             confirmed: bool
+     *         }
+     *     }
+     * } $context
+     */
+    public function __invoke(?object $item, array $context): Customer
```

PHPMD (2.15.0)

Avoid unused parameters such as '$item'. (undefined)

(UnusedFormalParameter)